python_modules/dagster-graphql/dagster_graphql/schema/partition_sets.py

```diff
@@ -121,6 +122,32 @@ class Meta:
         name = "PartitionStatusesOrError"
 
 
+class GraphenePartitionsStatus(graphene.Enum):
+    COMPLETED = "COMPLETED"
+    FAILED = "FAILED"
+    REQUESTED = "REQUESTED"
+
+    class Meta:
+        name = "AssetPartitionsStatus"
+
+
+class GraphenePartitionsStatusCounts(graphene.ObjectType):
+    partitionsStatus = graphene.NonNull(GraphenePartitionsStatus)
+    count = graphene.NonNull(graphene.Int)
+
+    class Meta:
+        name = "PartitionBulkActionStatusCounts"
+
+
+class GrapheneAssetPartitionsStatusCounts(graphene.ObjectType):
+    class Meta:
+        name = "assetPartitionsStatusCounts"
+
+    assetKey = graphene.NonNull(GrapheneAssetKey)
+    numPartitionsTargeted = graphene.NonNull(graphene.Int)
+    partitionsStatusCounts = non_null_list(GraphenePartitionsStatusCounts)
```

Comment:
Rather than introduce `GraphenePartitionsStatusCounts` and `GraphenePartitionsStatus`, are there drawbacks to just doing:
```python
numPartitionsRequested = graphene.NonNull(graphene.Int)
numPartitionsCompleted = graphene.NonNull(graphene.Int)
numPartitionsFailed = graphene.NonNull(graphene.Int)
```

Reply:
i don't think so, probably cleaner to have it in individual resolvers as you suggested